### PR TITLE
Feature/103 table to show needs ive created but left unassigned

### PR DESCRIPTION
**Pull request by jamiebuckley**
Currently this is implemented as another dashboard link in the sidebar
Screenshot from 2020-05-12 09-50-42
There are designs that display this embedded in a page, but as it is part of a wider dashboard redesign, that seems better left until those are finalized

### CODE REVIEW

**Comment by jamiebuckley** [comment]
[Table to show needs I've created but left 'unassigned'](https://trello.com/c/1lkaPsQY/103-table-to-show-needs-ive-created-but-left-unassigned)

File: features/list_support_actions.feature [hunk 1]
```diff
@@ -46,4 +46,20 @@ Feature: List support actions
...
+  Scenario: See unassigned needs I have created in the created and unassigned list
+    Given a unique resident
+    And I add support actions "Dog walking"
+    And I submit the add support actions form
+    When I go the the created and unassigned support action list
+    Then I see the support action for category "Dog walking" in the results
+
+  @javascript
+  Scenario: Do not see assigned needs I have created in the created and unassigned list
+    Given a unique resident
```
Comment (rik86future) [inline features/list_support_actions.feature:60]: You could compress this into "Given I have created a support action "Dog Walking""
Which calls
```
step  a unique resident
step I add support actions "Dog walking"
step I submit the add support actions form
step I assign the support action to another user
```

It does not make any difference expect for it reading a bit easier

File: app/controllers/needs_controller.rb [hunk 2]
```diff
@@ -8,14 +8,20 @@ class NeedsController < ApplicationController
   helper_method :get_param
 
   def index
-    @params = params.permit(:assigned_to, :status, :category, :page, :order_dir, :order, :commit, :is_urgent)
+    @params = params.permit(:assigned_to, :status, :category, :page, :order_dir, :order, :commit, :is_urgent, :created_by_me)
     @users = User.all
```
Comment (rik86future) [inline app/controllers/needs_controller.rb:12]: I am not sure if we still need there @users and @roles variables here

Comment (jamiebuckley) [reply app/controllers/needs_controller.rb]: Good spot - these have been superceded by the 'construct_assigned_to_options' method, will remove them

File: features/list_support_actions.feature (hunk 1 above)
Comment (jamiebuckley) [reply features/list_support_actions.feature]: Fair point, have updated

**Pull request merged**